YourName/YourName/Sources/Common/Extensions/ViewController.swift

```diff
@@ -22,6 +22,10 @@ class ViewController: UIViewController {
         #endif
     }
     
+    deinit {
+        print("\(String(describing: self)) deinit")
```

Comment:
```suggestion
        print("💀 \(String(describing: self)) deinit")
```

이렇게 이모지들 이용해서 더 눈에 띄게해볼까여?!